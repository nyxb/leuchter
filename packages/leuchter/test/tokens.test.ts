import { expect, test } from 'vitest'
import { highlight } from '..'

test('remove empty space tokens', async () => {
   const code = `export default   function Gallery() {

  }`
   const result = await highlight(code, 'js', 'github-dark')
   expect(result.lines).toMatchSnapshot()
})
