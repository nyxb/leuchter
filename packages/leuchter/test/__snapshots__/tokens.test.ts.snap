// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`remove empty space tokens 1`] = `
[
  [
    {
      "content": "export default   function ",
      "style": {
        "color": "#FF7B72",
      },
    },
    {
      "content": "Gallery",
      "style": {
        "color": "#D2A8FF",
      },
    },
    {
      "content": "() ",
      "style": {
        "color": "#FFA657",
      },
    },
    {
      "content": "{",
      "style": {
        "color": "#C9D1D9",
      },
    },
  ],
  [
    {
      "content": "",
      "style": {
        "color": "#C9D1D9",
      },
    },
  ],
  [
    {
      "content": "  }",
      "style": {
        "color": "#C9D1D9",
      },
    },
  ],
]
`;
